CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -10,20 +10,21 @@ function cleanup {
 # which can lead to errors.
 if [ -n "${UVCDAT_SETUP_PATH}" ] ; then
   if [ "${UVCDAT_SETUP_PATH}" = "${install_prefix}" ] ; then
+    echo "Nothing to do since UVCDAT is already setup at: ${UVCDAT_SETUP_PATH}" 1>&2
     cleanup
     return 0
   else
-    echo "ERROR: UVCDAT setup was previously sourced at \"${UVCDAT_SETUP_PATH}\"" 1>&2
-    echo "ERROR: There is no need to run setup_runtime manually anymore." 1>&2
-    echo "ERROR: Open a new shell in order to use a different install location." 1>&2
+    echo "INFO: UVCDAT setup was previously sourced at: ${UVCDAT_SETUP_PATH}" 1>&2
+    echo "INFO: There is no need to run setup_runtime manually anymore." 1>&2
+    echo "INFO: Open a new shell in order to use a different install location." 1>&2
```

Comment:
@remram44 what's important here? Users get the information right? whether you and I tag it as INFO or ERROR they will see the message. If you want we can change it to WARNING but its not a system error by any means in my opinion.